deploy/cluster-manager/config/rbac/cluster_role.yaml

```diff
@@ -55,3 +58,65 @@ rules:
 - apiGroups: ["migration.k8s.io"]
   resources: ["storageversionmigrations"]
   verbs: ["create", "get", "list", "update", "watch", "patch", "delete"]
+
+- apiGroups: ["addon.open-cluster-management.io"]
+  resources: ["managedclusteraddons"]
+  verbs: ["get", "list", "watch"]
```

Comment:
create/update/patch is also need for addon-manager controller

Reply:
added